src/agenda/reservation-list/index.js

```diff
@@ -179,7 +178,7 @@ class ReactComp extends Component {
 
   render() {
     if (!this.props.reservations || !this.props.reservations[this.props.selectedDay.toString('yyyy-MM-dd')]) {
-      return (<ActivityIndicator style={{marginTop: 80}}/>);
+      return this.props.renderEmptyData();
```

Comment:
I think you should set the `ActivityIndicator` that was here before as default, so there's no breaking change? 🤔

Reply:
yes, but you could still override it with something else than a spinner, but it should default to show the same ActivityIndicator as before, to not break other peoples apps 👍